apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,45 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Menuconfig Setup
+**For build configuration details refer to**
+'build/configs/rtl8721csm/security_hal_test_tz/defconfig'
+
+**Enable web server**
+> Networking support > Protocols > Webserver 
+
+> Application Configuration > Examples > Webserver example
+
+> Application Configuration > Examples > Webserver test
+
+## webserver_test usage :
+
+**webserver_test iterations method uri [options...]**
+
+- method and url parameters should be given,  [options] parameters are optional.
+- method is one if "GET", "POST", "PUT", "DELETE".  
+- url should start with "http://" or "https://".
+   
+[options] parameter support several different options.
+-    chunked=1        : Enable chunked encoding (default is disabled)
+-    test_entity=SIZE : Input test entity dummy data (default is 0)
+-    keep-alive=X     : Turns on keep-alive and sends X requests (default Keep-Alive is OFF)
+-    trailer=1        : Turns on Trailer-header
+
+Example usage:
+`webserver_test 3 POST https://127.0.0.1/ chunk-count=4 test_entity=5 trailer=1 keep-alive=2`
+
+Above command runs 3 iterations of the application
```

Comment:
Application is spawned only once from TASH
3 iterations of POST via https protocol is executed

Please check can above adds better clarity